SkietbaanBE/SkietbaanBE/appsettings.Development.json

```diff
@@ -6,5 +6,8 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "ConnectionStrings": {
+    "BloggingDatabase": "Server=dev.retrotest.co.za;Database=skietbaan;User Id=group3;Password=Y%6KG-bmr96Pe5rC;"
```

Comment:
Fix name,
Also you need to remove the values controleer as I dont htink it is code its template